src/orion/core/worker/primary_algo.py

```diff
@@ -137,48 +137,72 @@ def suggest(self, num: int) -> list[Trial] | None:
         New parameters must be compliant with the problem's domain `orion.algo.space.Space`.
 
         """
-        transformed_trials = self.algorithm.suggest(num)
-
-        if transformed_trials is None:
-            return None
 
         trials: list[Trial] = []
-        for transformed_trial in transformed_trials:
-            if transformed_trial not in self.transformed_space:
-                raise ValueError(
-                    f"Trial {transformed_trial.id} not contained in space:\n"
-                    f"Params: {transformed_trial.params}\n"
-                    f"Space: {self.transformed_space}"
-                )
-            original = self.transformed_space.reverse(transformed_trial)
-            if original in self.registry:
+        max_suggest_attempts = 100
```

Comment:
This should be configurable.

Reply:
As a constructor argument to the wrapper? Or parameter to `suggest` ?

Reply:
Made it an attribute in 872f094281e8e855f0bf5be05b11b7988131b228, is this sufficient, you think?

Reply:
Sorry for the delay. It's a bit tricky here since at the constructor level it would add an argument specific for the wrapper, should I don't think should be exposed to the users since the define the algo configuration directly, they are not even aware that there is a wrapper. Same for suggest, it should follow the same interface as the algorithms it is wrapping. Maybe your solution is the best one for now and we can reassess later on if needed.